internal/server/mcp/v20241105/types.go

```diff
@@ -136,3 +139,38 @@ type CallToolResult struct {
 	// If not set, this is assumed to be false (the call was successful).
 	IsError bool `json:"isError,omitempty"`
 }
+
+/* Prompts */
+
+// Sent from the client to request a list of prompts the server has.
+type ListPromptsRequest struct {
+	PaginatedRequest
+}
+
+// The server's response to a prompts/list request from the client.
+type ListPromptsResult struct {
+	PaginatedResult
+	Prompts []prompts.McpManifest `json:"prompts"`
+}
+
+// GetPromptRequest is sent from the client to get a specific prompt with substituted arguments.
+type GetPromptRequest struct {
+	jsonrpc.Request
+	Params struct {
+		Name      string         `json:"name"`
+		Arguments map[string]any `json:"arguments,omitempty"`
+	} `json:"params"`
+}
+
+// GetPromptResult is the server's response to a prompts/get request.
+type GetPromptResult struct {
+	jsonrpc.Result
+	Description string            `json:"description,omitempty"`
+	Messages    []ResponseMessage `json:"messages"`
+}
+
+// ResponseMessage represents a single message in the GetPromptResult.
+type ResponseMessage struct {
```

Comment:
```suggestion
type PromptMessage struct {
```

use the same name as the schema